conductor/lib/common.py

```diff
@@ -7,6 +7,7 @@
 import multiprocessing
 import os
 import platform
+from pprint import pformat
```

Comment:
`from x import y` go after `import x` statements.

eg. this is what `isort` gives me:
```
import base64
import datetime
import functools
import hashlib
import json
import logging
import multiprocessing
import os
import platform
import random
import signal
import subprocess
import sys
import time
import traceback
from pprint import pformat

import yaml
```

Reply:
We should probably decide as a team on how we want to sort imports.  I haven't seen much in google style guide or pep8 (aside from grouping imports by builtin, thirdparty, internal).
Personally, I sort imports alphabetically, considering the uppermost namespace (parent packages) when evaluating order. I don't distinguish/consider whether it's a  `from` vs `import`.  I find that it makes it easier for me to look something up, if there is one continuous alphabetical ordering, rather than having multiple sections of their own ordering.  But we're all weird humans, thinking in weird ways.

also, I noticed that isort has several different options that make dramatic differences  (`--no-sections`, `--order-by-type`,  `--project`, `--thirdparty` ,  etc).  Might be worth playing around with these to find a workflow something that we all like (dislike ;) ).

Reply:
Good call, let's make this a team decision.